.github/workflows/ld.yml

```diff
@@ -0,0 +1,108 @@
+name: ld
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*'
+    paths:
+      - 'src/ld/**'
+  pull_request:
+    branches:
+      - 'master'
```

Comment:
don't think we need this, similar to https://github.com/moby/buildkit/pull/3616

Reply:
Not part of functional changes in this PR. Just replicating what other workflows do.